salt/metalk8s/addons/ui/deployed/ui.sls.in

```diff
@@ -27,7 +27,7 @@ include:
 %}
 
 {%- set stripped_base_path = metalk8s_ui_config.spec.basePath.strip('/') %}
-{%- set normalized_base_path = '/' ~ stripped_base_path if stripped_base_path else '' %}
+{%- set normalized_base_path = '/' ~ stripped_base_path if stripped_base_path else '/' %}
```

Comment:
Maybe a stupid comment, but isn't this equivalent to

```jinja
{%- set normalized_base_path = '/' ~ stripped_base_path %}
```

?

Reply:
done, I squashed and took care of it at the same time